File: OpenTokFs.RequestTypes/OpenTokLayoutClassChangeRequest.cs
using Newtonsoft.Json;
using System.Collections.Generic;
using System.Linq;

namespace OpenTokFs.RequestTypes {
    public class OpenTokLayoutClassChangeRequest {
        [JsonProperty("items")]
        public IEnumerable<ClassChange> Items { get; set; } = Enumerable.Empty<ClassChange>();

        public class ClassChange {
            [JsonProperty("id")]
            public string Id { get; set; } = "";

            [JsonProperty("layoutClassList")]
            public IEnumerable<string> LayoutClassList { get; set; } = Enumerable.Empty<string>();
        }
    }
}

File: OpenTokFs/RequestTypes/OpenTokLayoutClassChangeRequest.fs
namespace OpenTokFs.RequestTypes

module OpenTokLayoutClassChangeRequest =
    type ClassChange() =
        member val id = "" with get, set
        member val layoutClassList: seq<string> = Seq.empty with get, set

type OpenTokLayoutClassChangeRequest() =
    member val items: seq<OpenTokLayoutClassChangeRequest.ClassChange> = Seq.empty with get, set
